app/Http/Controllers/MismatchController.php

```diff
@@ -46,14 +44,11 @@ public function index(MismatchesRequest $request)
      */
     public function update(MismatchesRequest $request, $id)
     {
-        $mismatch = Mismatch::find($id);
-
         // TODO: validate value of review_status
         // TODO: changes to any other property will lead to a validation error
-        
-        $mismatch->review_status = $request->review_status;
-        $mismatch->save();
-       
+        $mismatch = Mismatch::find($id);
+        $mismatch->update($request->all());
```

Comment:
I don't think this is what we want to do. We don't want to be able to modify all the parameters.